tests/result/test_qblox_result.py

```diff
@@ -1,377 +1,377 @@
-        """Test the array property of the QbloxResult class."""
-        array = qblox_result_scope.array
-        dummy_qrm.start_sequencer()
-        dummy_qrm.store_scope_acquisition(0, "single")
-        scope_data = dummy_qrm.get_acquisitions(0)["single"]["acquisition"]["scope"]
-        path0, path1 = scope_data["path0"]["data"], scope_data["path1"]["data"]
-        assert np.shape(array) == (2, 16380)  # I/Q values of the whole scope
-        assert np.allclose(array, np.array([path0, path1]))
-
-    def test_array_property_of_binned_data(self, dummy_qrm: Pulsar, qblox_result_noscope: QbloxResult):
-        """Test the array property of the QbloxResult class."""
-        array = qblox_result_noscope.array
-        assert np.shape(array) == (2, 1)  # (1 sequencer, I/Q, 1 bin)
-        dummy_qrm.start_sequencer(0)
-        dummy_qrm.store_scope_acquisition(0, "single")
-        bin_data = dummy_qrm.get_acquisitions(0)["single"]["acquisition"]["bins"]["integration"]
-        path0, path1 = bin_data["path0"], bin_data["path1"]
-        assert np.allclose(array, [path0, path1])
-
-    def test_array_property_asymmetric_bins_raise_error(self, qblox_asymmetric_bins_result: QbloxResult):
-        """Tests if IndexError exception is raised when sequencers have different number of bins.
-
-        Args:
-            qblox_asymmetric_bins_result (QbloxResult): QbloxResult instance with different number of bins on each sequencer.
-        """
-        bin_shape = [len(bin_s["bins"]["threshold"]) for bin_s in qblox_asymmetric_bins_result.qblox_raw_results]
-        with pytest.raises(
-            IndexError,
-            match=re.escape(
-                f"All measurements must have the same number of bins to return an array. Obtained {len(bin_shape)} measurements with {bin_shape} bins respectively."
-            ),
-        ):
-            _ = qblox_asymmetric_bins_result.array
-
-    def test_counts_error_multi_measurement(self, qblox_multi_m_results: QbloxResult):
-        """Test that an error is raised in counts if there is more than one result for a single qubit"""
-        with pytest.raises(
-            NotImplementedError, match="Counts for multiple measurements on a single qubit are not supported"
-        ):
-            _ = qblox_multi_m_results.counts()
-
-    def test_samples_error_multi_measurement(self, qblox_multi_m_results: QbloxResult):
-        """Test that an error is raised in counts if there is more than one result for a single qubit"""
-        with pytest.raises(
-            NotImplementedError, match="Samples for multiple measurements on a single qubit are not supported"
-        ):
-            _ = qblox_multi_m_results.samples()
-
-    def test_to_dataframe(self, qblox_result_noscope: QbloxResult):
-        """Test the to_dataframe method."""
-        dataframe = qblox_result_noscope.to_dataframe()
-        assert isinstance(dataframe, pd.DataFrame)
+# """ Test Results """
```

Comment:
Why is this file commented instead of deleted?

Reply:
Nice catch, I deleted it.